src/components/Activity/ActivityList/ActivityList.tsx

```diff
@@ -131,7 +136,7 @@ export const ActivityList = (props: ActivityListProps) => {
 const Search = ({ send, value }: { value?: string; send: (search?: string) => void }) => {
   const handler = useCallback(debounce(send), []);
   return (
-    <div className="relative block">
+    <div className="foo relative block">
```

Comment:
Will allow it, but we should remove this `foo` class its clearly some sort of debugging...